bp/src/BPXCTestFile.m

```diff
@@ -221,4 +219,7 @@ - (id)copyWithZone:(NSZone *)zone {
     return copy;
 }
 
+- (BOOL)isEqual:(id)object{
+    return [self.name isEqualToString: ((BPXCTestFile*)object).name];
```

Comment:
test object type first. if it's not the same return false, then cast and compare

Reply:
Done.